content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -961,5 +961,12 @@ The following is an example procedure to upgrade from `v1beta1` to `v1`.
     the same content. This forces the backend to write objects in the current
     storage version, which is `v1`.
 2. Remove `v1beta1` from the CustomResourceDefinition `status.storedVersions` field.
+This can be done by accessing kubernetes api with patch request
+```bash
+curl -d '[{ "op": "replace", "path":"/status/storedVersions", "value": ["v1"] }]' \
+  -H "Content-Type: application/json-patch+json" \
+  -X PATCH \
+ http://localhost:8080/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/$resource_name/status
```

Comment:
Possibly:
```suggestion
 https://api-server-hostname.example/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/$resource_name/status
```
?

HTTPS because it's a very good idea to use TLS to protect this traffic.

Reply:
Explaining this in curl terms looks far too detailed here. It's one way out of a hundred other ways. You can use `kubectl edit`, you can use some programmed mechanism in an installer with client-go.

In general: the v1beta1 API for CRDs is replaced with v1 by now. Don't use v1beta1 in new documentation.

Reply:
Thank you all for the feedback. @sttts I wasn't able to achive that using `kubectl edit`. 

I have spend quite some time trying to remove old stored version manually and think it might be useful to describe it. The only ways I know of modifying `storedVersion` are api and go-client. I can describe those two in separate sub-section. Maybe this will be a better idea?

Reply:
> I wasn't able to achive that using `kubectl edit`

If it is possible, one for the [cheat sheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/) perhaps?

Reply:
I've descirbed all things I've tried and what worked in this issue. https://github.com/kubernetes/website/issues/18248#issuecomment-579713447

Reply:
We are on it debugging what's going on with kubectl edit. There is a bug. Please hold back merging this doc PR until we know. /cc @vareti

Reply:
Based on https://github.com/kubernetes/website/pull/18926#discussion_r374549797:
/hold

Reply:
If the resources can be modified using any of the available channels this should not be part of this documentation page. When the issue is resolved maybe it's still a good idea to include some examples how the resources can be modified?

Reply:
@sttts is the /hold still valid?

Reply:
I understand that kubectl does not work because it cannot access the status subresource. I would like to see at least a clear explanation of that fact. The curl line should follow best practices: https, version v1, maybe via  `kubectl proxy`.

Reply:
You're right. I'll update the pull request. Where can I find description of best practises I should follow?